### DISCUSSION TITLE
Help sorting out Objects are not valid as a React child error message

### DISCUSSION THREAD

**Discussion by Isaac-Tait** [tinacms/tinacms#4192] [Help]
I have the following in `blog > index.js` file:

```
<div className="p-2">
            {postsList.map((post) => (
              <div
                key={post.node.id}
                className="my-4 underline hover:text-white"
              >
                <Link href={`/news/${post.node._sys.filename}`}>
                  {post.node.title}
                </Link>
              </div>
            ))}
 </div>
```

Originally `post.node.title` was `node. post._sys.filename` but I wanted it to be the title, not the filename - so I changed it. No problems there.

Then I added an excerpt field in `tina > config.js` 

```
{
            type: "rich-text",
            name: "excerpt",
            label: "Excerpt",
            isExcerpt: true,
            required: true,
},
```
When I tried to change `post.node.title` to `post.node.excerpt` in the `index.js` file I got the following error:

```
Server Error
Error: Objects are not valid as a React child (found: object with keys {type, children}). If you meant to render a collection of children, use an array instead.
```

Why is Tina able to change to title but not the new schema of excerpt? 

I am asking because eventually I would like to be able to add the blog post's excerpt below the blog post link but I keep getting errors.

**Comment by mAAdhaTTah** [comment]
You have to pass rich text into `TinaMarkdown`. See here: https://tina.io/docs/editing/markdown/#rendering-content-with-object-object
